### DISCUSSION TITLE
Sync Not Working Last Few Days

### DISCUSSION THREAD

**Discussion by gunjanmarwah** [philosowaffle/peloton-to-garmin#530] [Q&A]
All of a sudden, I'm not getting anything through, even with manual jobs. Everything says successful but it is not actually sending any information 😞

**Comment by philosowaffle** [comment]
Probably same issue as #526

**Reply by gunjanmarwah** [reply]
@philosowaffle sorry I don't even know how to interpret that post. I am not a developer and this is the only thing I use Git for. Your setup instructions were very clear so I was able to get it going but I have zero idea how to troubleshoot. Are you able to give me any further guidance on how to check if it is indeed the same? I can't find any details on an there being an error at all, let alone any specific error message. Thanks!

**Reply by philosowaffle** [reply]
Ok, I found what you're doing, you're using the [github action](https://philosowaffle.github.io/peloton-to-garmin/install/github-action.html) to do your sync.  The way to view logs is this:

1. Go to your copy of the [repository](https://github.com/gunjanmarwah/peloton-to-garmin)
2. Select the `Actions` tab, along the top left of the view, it will land you [here](https://github.com/gunjanmarwah/peloton-to-garmin/actions)
3. Click the very first `Sync workflow` item to open up that [workflow](https://github.com/gunjanmarwah/peloton-to-garmin/actions/runs/6333161176)
4. In the left hand menu, under `Jobs` click `sync` which will land you [here](https://github.com/gunjanmarwah/peloton-to-garmin/actions/runs/6333161176/job/17200854912)
5. Finally, expand the item that says `Run /app/PelotonToGarminConsole`

Now we can see the logs that were generated as part of that sync attempt.  And I a can see the same error is indeed happening for you as is happening for other folks.

Regarding the error, I have identified what needs to be done to fix the issue, however I won't have time to work on the fix until this weekend.  When the fix is done I will update issue #526 and be sure to add instructions for how to get the latest version of P2G with the fix in it.

**Comment by gunjanmarwah** [comment]
Thank you so much for that detailed info and sharing your game plan! Really
appreciate you!
